mteb/cache.py

```diff
@@ -214,3 +223,249 @@ def clear_cache(self) -> None:
 
     def __repr__(self) -> str:
         return f"ResultCache(cache_path={self.cache_path})"
+
+    def get_cache_paths(
+        self,
+        models: Sequence[str] | Sequence[ModelMeta] | None = None,
+        tasks: Sequence[str] | Sequence[AbsTask] | None = None,
+        require_model_meta: bool = True,
+        include_remote: bool = True,
+    ) -> list[Path]:
+        """Get all paths in the cache directory.
```

Comment:
All "paths" = all paths to result json files? Let's specify that.
Maybe mention why & how this method is useful? Also include an example of what the output looks like?
```suggestion
        """Get all paths to result JSON files in the cache directory. These paths can then be used to fetch task results, like:
        ```
        for path in paths:
            task_result = TaskResult.from_disk(path)
        ```
```